lib/cdo/mailjet.rb

```diff
@@ -16,7 +18,8 @@ def self.enabled?
     DCDO.get('use_mailjet', false) &&
       !Rails.env.test? &&
       API_KEY.present? &&
-      SECRET_KEY.present?
+      SECRET_KEY.present? &&
+      MAILGUN_API_KEY.present?
```

Comment:
Now when I see api keys I'm called to ask: have we tested that the key is correctly configured? I'm sorry if this has already been tested elsewhere and I'm late to the game!